python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -0,0 +1,280 @@
+# This file is part of meas_algorithms.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Collection of small images (stamps), each centered on a bright star.
+"""
+
+import collections.abc
+import numpy as np
+
+from lsst import afw
+from lsst import geom
```

Comment:
because you are using very few symbols, its probably worth the saved characters later to do
```python
from lsst.geom import Box2I, Point2I, Extent2I
```